### PR TITLE
Document cosmosdb database type in fluffy

### LINKED ISSUE #7947: Document cosmosdb database type in fluffy

### PR DESCRIPTION
**Pull request by yonipeleg33**
Closes #7947

### CODE REVIEW

**Review by Isan-Rivkin — APPROVED** [review]
LGTM, nice change

File: docs/enterprise/fluffy-configuration.md [hunk 1]
```diff
@@ -54,11 +54,21 @@ This reference uses `.` to denote the nesting of values.
...
+    + `database.cosmosdb.database` `(string : "")` - CosmosDB database name.
+    + `database.cosmosdb.container` `(string : "")` - CosmosDB container name.
+    + `database.cosmosdb.throughput` `(int32 : )` - CosmosDB container's RU/s. If not set - the default CosmosDB container throughput is used.
+    + `database.cosmosdb.autoscale` `(bool : false)` - If set, CosmosDB container throughput is autoscaled (See CosmosDB docs for minimum throughput requirement). Otherwise, uses "Manual" mode ([Docs](https://learn.microsoft.com/en-us/azure/cosmos-db/provision-throughput-autoscale)).
+
+      + `database.local` - Configuration section when using `database.type="local"`
+        + `database.local.path` `(string : "~/fluffy/metadata")` - Local path on the filesystem to store embedded KV metadata
+        + `database.local.sync_writes` `(bool: true)` - Ensure each write is written to the disk. Disable to increase performance
+        + `database.local.prefetch_size` `(int: 256)` - How many items to prefetch when iterating over embedded KV records
+        + `database.local.enable_logging` `(bool: false)` - Enable trace logging for local driver
```
[the remark this answers is not in the document]
Comment (yonipeleg33) [reply docs/enterprise/fluffy-configuration.md:71]: I kept the same order as in [configuration.md](https://github.com/treeverse/lakeFS/blob/master/docs/reference/configuration.md#database)

[the remark this answers is not in the document]
Comment (Isan-Rivkin) [reply docs/enterprise/fluffy-configuration.md:71]: Consistent bugs are better than inconsistent features :P 
Joke, it's better to be consistent, 100 👍

[the remark this answers is not in the document]
Comment (Isan-Rivkin) [reply docs/enterprise/fluffy-configuration.md:71]: Consistent bugs are better than inconsistent features :P 
Joke, it's better to be consistent, 100 👍

**Review by Isan-Rivkin — APPROVED** [review]

**Pull request merged**